Sources/CGFloat.swift

```diff
@@ -1,10 +1 @@
-//
-//  CGFloat.swift
-//  UIKit
-//
-//  Created by Chris on 19.06.17.
-//  Copyright © 2017 flowkey. All rights reserved.
-//
-
-import struct Foundation.CGFloat
-public typealias CGFloat = Foundation.CGFloat
+public typealias CGFloat = Double
```

Comment:
since https://github.com/apple/swift-evolution/blob/main/proposals/0307-allow-interchangeable-use-of-double-cgfloat-types.md, trying to mimic the original behaviour does more harm than good. It is widely accepted that `Double` is a better type to use by default, even on arm32.